examples/webgl_loader_collada_kinematics.html

```diff
@@ -120,14 +120,10 @@
 
 				// Lights
 
-				var directionalLight = new THREE.HemisphereLight( 0xffeeee, 0x111122 );
-				directionalLight.position.x = Math.random() - 0.5;
```

Comment:
The camera is rotating, the point light is rotating, the model is rotating. I did not think we need the hemisphere light to have a random direction. The hemispere light is to add ambient light only -- brighter on the upper half and darker on the lower half.